app/helpers/candidates_helper.rb

```diff
@@ -123,10 +123,10 @@ def placement_changed_text(event)
         "requalified the candidate on #{position_link}"
       when "reserved"
         "reserved the candidate on #{position_link}"
-      else
+      when "disqualified"
```

Comment:
these changes will raise error until all eventa are migrated, need to support both values: current and "disqualified" until all events are migrated